NashTeamAlpha/BangazonWeb
# New Product Form and database fix

## Pull Request Description

The code here creates a form that creates a new product. 
The Product Model needed to be changed to contain a SubProductTypeId rather than a ProductTypeId to comply with the ERD.
## Steps to Test

Include the applicable information from the list below here
1. Switch to `ddw-product-list`
2. Run `dotnet ef database update`
3. Make sure there are no error when creating a database
## Link to Feature Ticket
#47, #97
